Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use-Case-2_Two-Qubit-Randomized-Benchmarking/README.md

```diff
@@ -0,0 +1,118 @@
+
+## Two-Qubit Gate (CZ)
+The use-case is designed for flux-tunable transmon qubits where the qubit-qubit interaction is realized with a direct capacitive coupling. Utilizing this architecture it is possible to realize a flux-tuned ∣11⟩-∣02⟩. phase gate. An applied flux pulse that tunes the qubits in and out of the ∣11⟩-∣02⟩ avoided-crossing leads to a conditional phase accumulation. Leaving the system at the avoided-crossing for a specific time maps the state ∣11⟩ back into itself but acquires a minus sign in the process. As the computational states are far from being resonant with other transitions their phases evolve trivially and can be corrected using single qubit phase corrections and thus realize the CZ gate. The *baker.play* statement therefore contains a flux pulse that frequency-tunes transmon *q0* in and out of the avoided crossing ∣11⟩-∣02⟩ , while the *baker.frame_rotation_2pi* statements correct the single qubit phases.
+
+```python
+def bake_cz(baker: Baking, q0, q1):
+    q0_xy_element = f"q{q0}_xy" #
+    q1_xy_element = f"q{q1}_xy"
+    q0_z_element = f"q{q0}_z"
+    baker.play("cz", q0_z_element)
+    baker.align()
+    baker.frame_rotation_2pi(qubit0_frame_update, q0_xy_element)
+    baker.frame_rotation_2pi(qubit1_frame_update, q1_xy_element)
+    baker.align()
+```
+
+## Initialization
+Before each circuit, it is important to implement a initialization protocol to reset the qubits to the ground state. In the example the *prep* function contains a single QUA command *wait* and is called before each circuit execution to assure that the initial state is set to ∣00⟩. The time inside the *wait* statement is chosen to be a multiple of the characteristic decay time of the qubits *T1* to leave enough time for the qubit to relax after it has been excited to the excited state ∣1⟩. If single shot readout is implemented, it is possible to use active feedback to reset the qubit to the ground state ∣0⟩ by sending a pi-pulse if the qubit was measured in the excited state ∣1⟩. 
+
+```python
+def prep():
+    wait(int(10*qubit_T1))  # thermal preparation in clock cycles (time = 10 x T1 x 4ns)
+    align()
+```
+
+## Measurement
+Finally, the user has to implement a measurement that is performed at the end of the random gate circuits. In this example we send two readout pulses simultaneously to both resonators that are coupled to the individual qubits and demodulate the signal. We assume that we optimized the readout such that all information is contained in the *I0* and *I1*. We then assign a True or False value to boolean QUA variables *state0* and *state1* and return the result, where False should be returned for state ∣0⟩ and True for state ∣1⟩. This measurement function returns four possible outcomes: ∣00⟩, ∣01⟩, ∣10⟩ & ∣11⟩.
```

Comment:
...is contained in the *I0* and *I1* quadratures.
Because of the italic font I didn't immediately get that it was the 'I' quadrature....